### PR TITLE
Add some perspective.

### PR DESCRIPTION
**Pull request by tzarc**
As per the title.

### CODE REVIEW

**Comment by drashna** [comment]
Might be worth adding this link too? 
https://github.com/kossnocorp/etiquette/blob/master/README.md

**Comment by tzarc** [comment]
> Might be worth adding this link too? [https://github.com/kossnocorp/etiquette/blob/master/README.md](https://github.com/kossnocorp/etiquette/blob/master/README.md?rgh-link-date=2022-06-13T19%3A34%3A36Z)

Done.

**Review by KarlK90 — APPROVED** [review]
LGTM!

File: utils.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ module.exports = {
...
+  volunteers: 'To gain some perspective, please have a read of these links:\n- [Don\'t be that open-source user, don\'t be me](https://jacobtomlinson.dev/posts/2022/dont-be-that-open-source-user-dont-be-me/)\n- [Open Source Maintainers Owe You Nothing](https://mikemcquaid.com/2018/03/19/open-source-maintainers-owe-you-nothing/)\m- [GitHub Etiquette](https://github.com/kossnocorp/etiquette/blob/master/README.md)',
```
Comment (yanfali) [inline utils.js:202]: for long pieces of text, backticks are often easier to maintain and read. Especially if you're going to be using a lot of \n

**Pull request merged**